### ISSUE TITLE
Why not publish it as a jekyll github pages or wiki pages

### ISSUE THREAD

**Issue by anantshri** [doridori/Android-Security-Reference#3]
Why not publish the entire notes as a jekyll github.io page or wiki entries. will remove the need of using octopress as that's browser specific. and without a proper readme based linkage it gets difficult to work with repository.

**Comment by doridori** [comment]
Good idea. Not a fan of Github Wiki as find it clunky to use but a github.io site maybe nice.

Why are you mentioning octopress?

**Comment by anantshri** [comment]
Ah, looks like i meant octotree plugin and typed octopress. 
Basically with github.io you will no longer need octotree to navigate around you can have a proper navigation tree.

**Comment by doridori** [comment]
Good idea - will do when I get some time :)

**Comment by dufferzafar** [comment]
Or maybe this could be compiled into a neat book (PDF) using Gitbook or something. That'll allow easy reading on mobile devices.
